packages/teams-js/src/public/stageView.ts

```diff
@@ -115,4 +115,39 @@ export namespace stageView {
   export function isSupported(): boolean {
     return ensureInitialized(runtime) && runtime.supports.stageView ? true : false;
   }
+
+  export namespace stageClose {
+    /**
+     * Closes the current stage view.
+     * @returns Promise that resolves or rejects with an error once the stage view is closed.
+     */
+    export function closeCurrentWindow(): Promise<void> {
+      return new Promise((resolve) => {
+        ensureInitialized(runtime, FrameContexts.stage);
+
+        if (!isSupported()) {
+          throw errorNotSupportedOnPlatform;
+        }
+
+        resolve(
+          sendAndHandleSdkError(
+            getApiVersionTag(stageViewTelemetryVersionNumber, ApiName.StageView_Close),
+            'stageView.closeCurrentWindow',
+          ),
+        );
+      });
+    }
+
+    /**
+     * Checks if stageView capability is supported by the host
```

Comment:
This comment is wrong, it should be something like: 
```suggestion
     * Checks if stageView.<new capability name> capability is supported by the host
```